content/creator/wearables-and-emotes/manage-collections/uploading-smart-wearables.md

```diff
@@ -0,0 +1,165 @@
+---
+
+After generating your smart wearable using the [SDK7](/creator/development-guide/sdk7/smart-wearables/), the next step is to upload it to the builder. This document explains the process of uploading, publishing and putting Smart Wearables up for sale.
+
+#### Uploading Your File
+
+Remember that you need to create a collection before you can upload your file. If you don’t know how to do that, check [Creating a Collection](/creator/wearables-and-emotes/manage-collections/creating-a-collection.md). To upload your smart wearable, just drag and drop the file on the **_New Item_** window or browse your computer. It will automatically detect if the file is a smart wearable. **Remember that the collection max file size is 2MB**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/01_new_item.png" width="400" />
+
+When you upload the file, you will be asked to upload a video showcase.
+
+#### Uploading Your Video Showcase
+
+This video showcase will be displayed in the marketplace for buyers to see how the Smart Wearable works.
+
+To upload your video showcase, just drag and drop a **.mp4** video file on the **_Upload a video for your smart wearable_** window or browse your computer. **Remember that the video max file size is 4MB and max duration is 15 seconds**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/01_new_video.png" width="400" />
+
+When you upload the video showcase, you will be asked to enter a name and define the rarity and the category. You can also add the thumbnail for the smart wearable.
+
+### **Rarity**
+
+Select the Rarity of your item.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/03_rarity.png" width="400"/>
+
+| Rarity    | Number of Items |
+| --------- | --------------- |
+| Unique    | 1               |
+| Mythic    | 10              |
+| Legendary | 100             |
+| Epic      | 1,000           |
+| Rare      | 5,000           |
+| Uncommon  | 10,000          |
+| Common    | 100,000         |
+
+### **Category**
+
+Wearables are organized into different categories, depending on what part of an avatar they modify. Select the appropriate category for your item.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/04_category.png" width="400"/>
+
+### **Custom Thumbnails**
+
+You can add your own custom thumbnail by clicking on the camera icon and browsing your computer. **The thumbnail must be 256px square .png file with transparent background.** Collections containing thumbnails without transparent backgrounds will not be accepted by the Curation Committee.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/06_thumbnail.png" width="400"/>
+
+{{< hint warning >}}
+⚠️ Having a good render of your wearable is crucial in making it more appealing to potential users in the marketplace. **It's important to avoid adding any graphics other than the wearable itself, because this may cause the curation committee to reject it.**
```

Comment:
```suggestion
⚠️ Having a good render of your wearable is crucial for making it more appealing to potential Marketplace users. **It's important to avoid adding graphics other than the wearable itself because this may cause the Curation Committee to reject it.**
```